bottery/platform/telegram.py

```diff
@@ -141,7 +141,8 @@ def build_message(self, data):
         if not view:
             return
 
-        response = view(message)
+        # TODO: Test if the view returned something or not
+        response = await self.get_response(view, message)
```

Comment:
It is a matter of style, but I think this reads nicely:

```python
response = await view(message) if inspect.iscoroutinefunction(view) else view(message)
```

Reply:
YeI think it's a good idea to have this if in a function (maybe `get_response` isn't the best name) because other engines could have the same logic without reimplementing the logic. I tried the one line expression, but it gets over 80 chars (🔥).